site/indexNew.php

```diff
@@ -44,25 +44,42 @@
 <body class="light">
 <div id="themeButton" onclick="toggleTheme()"></div>
 <div id="dataRequestBox"><p>Allow GM4 to remember your preferences?</p><span class="smallButton" onclick="storeUserPreferences()">Allow</span> <span class="smallButton" onclick="$(this).parent().hide();dataprompted = true;">Dismiss</span></div>
-<div id="header">
-  <h1><img src="images/logo/logo_clear.svg" width="40px" height="40px" alt="Gamemode 4 Logo"/> Gamemode 4</h1>
-  <ul>
-    <a href="https://www.gm4.co/"><li>Home</li></a>
-    <a href="https://www.gm4.co/server"><li>Server</li></a>
-    <a href="https://www.gm4.co/wiki"><li>Wiki</li></a>
-    
-  </ul>
-</div>
-<div id="slideshowContainer">
-  <div id="slideshowLeft" class="slideshowArrow" onclick="slideshow('left')"></div>
-  <div id="slideshowRight" class="slideshowArrow" onclick="slideshow()"></div>
-  <div id="slideshowSubContainer" class="cardContainer">
-    
+<div id="landingGroup" width="100%">
+  <div id="headerGroup">
+    <div id="header" style="display: flex; margin-top: 4px">
```

Comment:
I'm guessing they're here cos you're still testing and this is a draft but these styles should be moved to the stylesheet.